From b30f84e240bd60a23508797e33a4777cbaa50949 Mon Sep 17 00:00:00 2001
From: Karel Zak <kzak@redhat.com>
Date: Tue, 17 Dec 2019 10:53:47 +0100
Subject: [PATCH] tests: update lscpu test for RHEL8

The test is originally from upstream (~v2.35), but RHEL-8 uses older
version with a little different output -- RHEL-8 has no vulnerability
fields and it counts cache sizes  in different way.

Addresses: https://bugzilla.redhat.com/show_bug.cgi?id=1739443
Signed-off-by: Karel Zak <kzak@redhat.com>
---
 tests/expected/lscpu/lscpu-s390-nested-virt | 57 ++++++++++-----------
 1 file changed, 26 insertions(+), 31 deletions(-)

diff --git a/tests/expected/lscpu/lscpu-s390-nested-virt b/tests/expected/lscpu/lscpu-s390-nested-virt
index 2665fd8dc..a6ab04f0b 100644
--- a/tests/expected/lscpu/lscpu-s390-nested-virt
+++ b/tests/expected/lscpu/lscpu-s390-nested-virt
@@ -1,34 +1,29 @@
-CPU op-mode(s):                  32-bit, 64-bit
-CPU(s):                          2
-On-line CPU(s) list:             0,1
-Thread(s) per core:              1
-Core(s) per socket:              1
-Socket(s) per book:              1
-Book(s) per drawer:              1
-Drawer(s):                       2
-NUMA node(s):                    1
-Vendor ID:                       IBM/S390
-Machine type:                    2964
-CPU dynamic MHz:                 5000
-CPU static MHz:                  5000
-BogoMIPS:                        3033.00
-Hypervisor:                      KVM/Linux
-Hypervisor vendor:               KVM
-Virtualization type:             full
-Dispatching mode:                horizontal
-L1d cache:                       256 KiB
-L1i cache:                       192 KiB
-L2d cache:                       4 MiB
-L2i cache:                       4 MiB
-L3 cache:                        64 MiB
-L4 cache:                        480 MiB
-NUMA node0 CPU(s):               0,1
-Vulnerability L1tf:              Not affected
-Vulnerability Meltdown:          Not affected
-Vulnerability Spec store bypass: Not affected
-Vulnerability Spectre v1:        Mitigation; __user pointer sanitization
-Vulnerability Spectre v2:        Mitigation; execute trampolines
-Flags:                           esan3 zarch stfle msa ldisp eimm dfp edat etf3eh highgprs te vx
+CPU op-mode(s):      32-bit, 64-bit
+CPU(s):              2
+On-line CPU(s) list: 0,1
+Thread(s) per core:  1
+Core(s) per socket:  1
+Socket(s) per book:  1
+Book(s) per drawer:  1
+Drawer(s):           2
+NUMA node(s):        1
+Vendor ID:           IBM/S390
+Machine type:        2964
+CPU dynamic MHz:     5000
+CPU static MHz:      5000
+BogoMIPS:            3033.00
+Hypervisor:          KVM/Linux
+Hypervisor vendor:   KVM
+Virtualization type: full
+Dispatching mode:    horizontal
+L1d cache:           128K
+L1i cache:           96K
+L2d cache:           2048K
+L2i cache:           2048K
+L3 cache:            65536K
+L4 cache:            491520K
+NUMA node0 CPU(s):   0,1
+Flags:               esan3 zarch stfle msa ldisp eimm dfp edat etf3eh highgprs te vx
 
 # The following is the parsable format, which can be fed to other
 # programs. Each different item in every column has an unique ID
-- 
2.21.0

